tests/test_class_basics.rs

```diff
@@ -235,7 +235,10 @@ fn test_unsendable<T: PyClass + 'static>() -> PyResult<()> {
         // Accessing the value inside this thread should not panic
         let caught_panic =
             std::panic::catch_unwind(std::panic::AssertUnwindSafe(|| -> PyResult<_> {
-                assert_eq!(obj.as_ref(py).getattr("value")?.extract::<usize>()?, 5);
+                assert_eq!(
+                    obj.bind(py).as_any().getattr("value")?.extract::<usize>()?,
```

Comment:
You might not need the `.as_any()` bit here to make `.getattr()` available, thanks to deref:

```suggestion
                    obj.bind(py).getattr("value")?.extract::<usize>()?,
```